# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: accessible/tests/browser/e10s/browser.ini
# Commit: 41d5f7688395
# Full Hash: 41d5f76883953154d9a53e09f805e1ce15cb57c9
# Author: James Teh <jteh@mozilla.com>
# Date: 2020-10-29 15:52:12
# Description:
#   Bug 1673919: Make ParagraphBoundaryRule skip over OuterDocAccessibles. r=MarcoZ
#   
#   The browser test is really a crash test, but it's really difficult to test a11y APIs properly using the crashtest framework.
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D95107
# ==============================================================================

diff -r ff7124e703ef -r 41d5f7688395 accessible/tests/browser/e10s/browser.ini
--- a/accessible/tests/browser/e10s/browser.ini	Thu Oct 29 02:55:42 2020 +0000
+++ b/accessible/tests/browser/e10s/browser.ini	Thu Oct 29 06:04:33 2020 +0000
@@ -33,6 +33,9 @@
 [browser_events_textchange.js]
 [browser_events_vcchange.js]
 
+# Text tests
+[browser_text_paragraph_boundary.js]
+
 # Tree update tests
 [browser_treeupdate_ariadialog.js]
 [browser_treeupdate_ariaowns.js]